lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -614,6 +614,11 @@ def test_grades_csv(self):
         )
         response = self.client.get(url)
         self.assertEqual(response.status_code, 200)
+        self.assertEqual(
+            response.__getitem__('content-disposition'),
```

Comment:
Why `response.__getitem__(...)` instead of just `resposne['content-disposition']`?